cleanlab/datalab/issue_manager/label.py

```diff
@@ -118,9 +124,36 @@ def _reset(self) -> None:
 
     def find_issues(
         self,
-        pred_probs: np.ndarray,
+        pred_probs: Optional[npt.NDArray] = None,
+        features: Optional[npt.NDArray] = None,
         **kwargs,
     ) -> None:
+        """Find label issues in the datalab.
+
+        Parameters
+        ----------
+        pred_probs :
+            The predicted probabilities for each example.
+
+        features :
+            The features for each example.
+        """
+        if pred_probs is None:
+            if features is None:
+                raise ValueError(
+                    "Either pred_probs or features must be provided to find label issues."
+                )
+            # produce out-of-sample pred_probs from features
+            knn = KNeighborsClassifier(n_neighbors=self.k)
+            knn.fit(features, self.datalab.labels)
+            pred_probs = knn.predict_proba(features)
+
+            encoder = OneHotEncoder(sparse_output=False)
```

Comment:
```suggestion
            encoder = OneHotEncoder()
```

This `sparse_output` arg is causing the unittests to fail for `python 3.7` as this arg was named something different in `sklearn==1.0` (which is the latest version supported on `python 3.7`, ref [here](https://scikit-learn.org/1.0/modules/generated/sklearn.preprocessing.OneHotEncoder.html)).

I also believe it is not required to force a non-sparse matrix here as the multiplication below will work just fine, so could just remove entirely.